common/test/acceptance/pages/studio/settings_certificates.py

```diff
@@ -290,21 +296,11 @@ def click_certificate_details_toggle(self):
         """
         self.find_css('a.detail-toggle').first.click()
 
-    ################
-    # Workflows
-    ################
-
-    def delete_certificate(self):
+    def click_certificate_delete_button(self):
         """
-        Delete the certificate
+        Click the Delete button (trash can icon) for the first certificate in the set
         """
-        self.wait_for_certificate_delete_button()
-
         self.find_css('.actions .delete').first.click()
```

Comment:
This would be my 'must' in this PR. If something happens on the page after clicking delete (arguably in all cases---successful or unsuccessful), it should waited on here.